plan/plan.go

```diff
@@ -100,6 +109,17 @@ func (p *Plan) Calculate() *Plan {
 	return plan
 }
 
+func targetChanged(desired, current *endpoint.Endpoint) bool {
+	return desired.Target != current.Target
+}
+
+func shouldUpdateTTL(desired, current *endpoint.Endpoint) bool {
+	if !desired.RecordTTL.IsConfigured {
+		return false
+	}
+	return desired.RecordTTL.Value != current.RecordTTL.Value
```

Comment:
this function would be just:

    return desired.RecordTTL != current.RecordTTL